### DISCUSSION TITLE
Is there any way to make u undo to behave like Sublime Text's Undo?

### DISCUSSION THREAD

**Discussion by saneef** [NeoVintageous/NeoVintageous#895] [Q&A]
Here is how the native Sublime Text Undo behaves

https://github.com/NeoVintageous/NeoVintageous/assets/81863/06cfad54-c83e-47e0-8464-a1fcb77d048e

But, when I'm using the Vintageous's undo `u` it deletes a group of changes. 

https://github.com/NeoVintageous/NeoVintageous/assets/81863/82f4216f-d8a3-4517-93cc-c9814e776fc1

I want to undo one change at a time. Is there any way to make `u` to go back one change at a time?

**Comment by gerardroche** [comment]
This is the intended behaviour. It's the way vim works. It groups all the insert changes into one undo redo i.e. `u CTRL-R`. I don't know of any way in vim to do one undo change at a time. 

There is `U` which will undo all changes on a single line but it's not implemented yet. That would probably mostly solve this issue. #258

I think it might be possible to add a setting to allow to disable the grouping of insert changes. Funnily enough, I'm actually working on refactoring this exact part  of the codebase. I'll keep this in mind while I'm refactoring and see if it's doable.

**Reply by saneef** [reply]
Thank you!

I tried to remap the key `u` to Sublime Text's `undo` command while in command mode. Because of the grouping, it also behaves like vim undo. I hope you'll be able to come up with some idea to disable the grouping.

**Comment by gerardroche** [comment]
Yes, mapping won't work because when you enter insert mode we begin grouping undo statements until you exit insert mode.  I'm pretty sure I can add the option to disable that feature.
